objc/runtime.h

```diff
@@ -289,7 +295,7 @@ Protocol *__unsafe_unretained* class_copyProtocolList(Class cls, unsigned int *o
 /**
  * Creates an instance of this class, allocating memory using malloc.
  */
-OBJC_PUBLIC
+OBJC_PUBLIC OBJC_RETURNS_RETAINED
 id class_createInstance(Class cls, size_t extraBytes);
```

Comment:
Not sure if it matters, but in Apple headers the attribute comes after the function instead of before.